bin/copyProjectDataToPrm.sh

```diff
@@ -137,11 +133,12 @@ function rsyncProjectRun() {
 	# ToDo: Do we need to add --delete to get rid of files that should no longer be there 
 	#       if an analysis run got updated?
 	#
-	local _transferSoFarSoGood='true'
+	local _transferSoFarSoGood
+	_transferSoFarSoGood='true'
 	echo "working on ${_project}/${_run}" > "${lockFile}"
 	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Rsyncing ${_project}/${_run} dir ..." \
 	2>&1 | tee -a "${JOB_CONTROLE_FILE_BASE}.started" 
-	rsync -av --progress --log-file="${JOB_CONTROLE_FILE_BASE}.started" --chmod='Du=rwx,Dg=rsx,Fu=rw,Fg=r,o-rwx' ${dryrun:-} \
+	rsync -av --progress --log-file="${JOB_CONTROLE_FILE_BASE}.started" --chmod='Du=rwx,Dg=rsx,Fu=rw,Fg=r,o-rwx' "${dryrun:-}" \
```

Comment:
Dit gaat mis. In principe natuurlijk goed dat er quotes om `${dryrun:-}` gaan, maar als die variabele bewust leeg is omdat je geen dryrun wil dan krijg je `""` en dat wordt door rsync als een relatief pad gezien en geeft dan hele andere resultaten; die lege string is natuurlijk geen bestaand bestand noch map, maar aangezien het resursief gaat en het een relatief pad is rsync je dan alles realtief t.o.v. de working dir :o. In `copyRawDataToPrm.sh` staat een voorbeeld van hoe dat op te lossen is: voeg de dryrun en --progress opties samen tot
```
rsync ..... "${dryrun:---progress}" .....
```
Dan krijg je of de waarde van ${dryrun}: `-n`
Of je krijgt als ${dryrun} leeg is: `--progress`.
Die twee zijn dan "mutually exclusive" en dat is Ok, want `--progress` is nogal loos bij een dryrun.

Reply:
ik had de quotes ook afgekeken van copyRawDataToPrm en had idd de --progress erachter zien staan maar wist niet zo goed wat de functie precies deed, bedankt voor de uitleg.. het is inmiddels aangepast